polkadot/primitives/src/v6/mod.rs

```diff
@@ -1783,24 +1783,27 @@ impl<T: Encode> WellKnownKey<T> {
 
 /// Type discriminator for PVF preparation timeouts
 #[derive(Encode, Decode, TypeInfo, Clone, Copy, Debug, PartialEq, Eq, Serialize, Deserialize)]
-pub enum PvfPrepTimeoutKind {
+pub enum PvfPrepKind {
 	/// For prechecking requests, the time period after which the preparation worker is considered
 	/// unresponsive and will be killed.
 	Precheck,
 
 	/// For execution and heads-up requests, the time period after which the preparation worker is
 	/// considered unresponsive and will be killed. More lenient than the timeout for prechecking
 	/// to prevent honest validators from timing out on valid PVFs.
-	Lenient,
+	Prepare,
 }
 
-/// Type discriminator for PVF execution timeouts
+/// Type discriminator for PVF execution timeouts and retries
 #[derive(Encode, Decode, TypeInfo, Clone, Copy, Debug, PartialEq, Eq, Serialize, Deserialize)]
-pub enum PvfExecTimeoutKind {
+pub enum PvfExecKind {
 	/// The amount of time to spend on execution during backing.
```

Comment:
I think this comment is old.

Reply:
Good question. :) It's not specific to timeouts anymore, so instead of "the amount of time", maybe it could just say "An execution request from Backing." or something like that.

Reply:
And can the docs be moved to `fn pvf_exec_timeout`?